### PR TITLE
minor typos and style changes

### PR DESCRIPTION
**Pull request by al-x**
##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/config/base.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Copyright (c) 2017 Ansible Project
+# Copyright (c) 2019 Ansible Project
```
Comment (samccann) [inline lib/ansible/config/base.yml]: Please don't change the copyright date.  The rest looks good and we can merge once you remove this edit.

Comment (acozine) [reply lib/ansible/config/base.yml]: For more info on the copyright question, see https://github.com/ansible/ansible/issues/45989#issuecomment-423635622.

Comment (al-x) [reply lib/ansible/config/base.yml]: Year reverted. Please review.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Thanks @al-x for the Ansible documentation fix!

**Pull request merged**